filelist.txt

```diff
@@ -48,6 +48,8 @@ file /etc/gshadow etc/gshadow 644 0 0
 file /etc/fstab etc/fstab 644 0 0
 file /etc/hosts etc/hosts 644 0 0
 slink /etc/mtab /proc/mounts 755 0 0
+dir /etc/udhcp 755 0 0
+file /etc/udhcp/simple.script stage/etc/udhcp/simple.script 755 0 0
```

Comment:
Should it really say **stage**/etc/udhcp/simple.script ... ? Everything else here refers directly to `etc/foo/bar`?